#162414128 like and dislike article reducers

tosinibrahim96 opened the pull request:
**What does this PR do?**
Implement reducers for article ReactionIcons

**Description of Task to be completed?**
- Create reducer file
- Add initial state
- Update state with database response
- Update view(Component) with new data

**Screenshot**
![satupload](https://user-images.githubusercontent.com/22124878/53679725-b0e25d80-3cd0-11e9-9caa-515f234ade7a.gif)

Commit:
Implement actions for liking and disliking an article

Commit:
Implement reducers for article ReactionIcons

- Create reducer file
- Add initial state
- Update state with database response
- Update view(Component) with new data

codrex reviewed (commented):
This PR is similar to this https://github.com/andela/krypton-ah-frontend/pull/68

Commit:
Implement reducers for article ReactionIcons

- Create reducer file
- Add initial state
- Update state with database response
- Update view(Component) with new data

Commit:
before final demo
